### DISCUSSION TITLE
在路由器上装了easytier，如何让局域网内的其他机器可以直接访问peer？

### DISCUSSION THREAD

**Discussion by CarterCL** [EasyTier/EasyTier#480] [Q&A]
看了下文档的“子网代理”，按我的理解应该是在路由器(192.168.0.0/24)开启easytier(10.0.0.1/24)并打开子网代理后，其他peer就可以直接通过192.168.0.x来访问局域网设备了。  

我想在路由器上运行easytier，然后局域网的设备可以直接通过10.0.0.x来访问所有的peer，应该怎么操作？

**Comment by KKRainbow** [comment]
参考网对网

**Reply by CarterCL** [reply]
感谢回复，我仔细看了下文档，是必须要开启子网代理是吗？    
还有个问题，假设两个网的局域网都是 192.168.0.0/24 网段，参照网对网配置之后，会发生冲突吗？

**Reply by 710850609** [reply]
我觉得会，要网段映射
<img width="1440" height="596" alt="image" src="https://github.com/user-attachments/assets/b6fcb7cc-6a67-4503-883c-df7855e7fd62" />
